packages/suite/src/views/wallet/nfts/ShownNfts.tsx

```diff
@@ -0,0 +1,32 @@
+import { Column } from '@trezor/components';
+import { SelectedAccountLoaded } from '@suite-common/wallet-types';
+
+import NftsTable from './NftsTable/NftsTable';
+
+type ShownNftsProps = {
+    selectedAccount: SelectedAccountLoaded;
+    searchQuery: string;
+};
+
+const ShownNfts = ({ selectedAccount, searchQuery }: ShownNftsProps) => {
+    return (
+        <Column gap={24}>
+            <NftsTable
+                selectedAccount={selectedAccount}
+                searchQuery={searchQuery}
+                type="ERC721"
+                shown={true}
```

Comment:
```suggestion
                isShown={true}
```
Maybe also something like `isVerifiedTokenSection`, `isShownTokenSection` would be more understandable